README.md

```diff
@@ -104,7 +109,7 @@ Training the 12B param model is not recommended on A10s.
 To train the 6.9B param model on A10 instances (ex: `g5.24xlarge`, 4 x A10 24GB; `Standard_NV72ads_A10_v5`, 2 x A10), make the following changes:
 
 - Set `per-device-train-batch-size` and `per-device-eval-batch-size` to 3 in the `train_dolly.py` invocation of `deepspeed`
-- Modify the deepspeed config file `ds_z3_bf16_config.json` to configure optimizer offload. Within the `"zero_optimization"` section, add:
+- Modify the deepspeed config file `a100_config.json` to configure optimizer offload. Within the `"zero_optimization"` section, add:
```

Comment:
maybe we can call this a100_a10_config

Reply:
Sure - we can always split these out if we get to having different configs for A10 and A100